web/cobrands/fixmystreet/staff.js

```diff
@@ -342,3 +342,29 @@ $.extend(fixmystreet.set_up, {
     });
   }
 });
+
+fixmystreet.hooks = {
```

Comment:
Could this set hooks if it doesn't exist, and then extend it? That way if more hooks are added in another file, nothing needs to be changed here. See how the existing `fixmystreet.set_up` does it in fixmystreet.js, might as well be consistent.